resources/assets/less/bem/osu-md.less

```diff
@@ -307,4 +309,8 @@
       text-align: right;
     }
   }
+
+  > *:last-child {
+    margin-bottom: 0;
+  }
```

Comment:
this breaks styling for store page (and not needed to be moved if using variable).